setup.py

```diff
@@ -11,29 +11,31 @@
 
 requirements = [
     'Click>=6.0',
-    # TODO: put package requirements here
+    'bigchaindb-common>=0.0.3',
+    'graphviz'
+]
```

Comment:
Might be nice to duplicate these across to `requirements_dev.txt`, or have `requirements_dev.txt` install from `setup.py`.